esx_service/vmdk_ops_test.py

```diff
@@ -715,20 +716,27 @@ def create_default_tenant_and_privileges(self):
         """ Create default tenant and privilege if not exist"""
 
         # create DEFAULT tenant if needed
-        error_info, tenant_uuid, tenant_name = auth.get_default_tenant()
+        error_msg, tenant_uuid, tenant_name = auth.get_default_tenant()
+        if error_msg:
+            logging.warning(error_msg)
+            return
+
         if not tenant_uuid:
             logging.debug("create_default_tenant_and_privileges: create DEFAULT tenant")
             error_info, tenant = auth_api._tenant_create(
                                            name=auth.DEFAULT_TENANT,
                                            description="This is a default tenant",
                                            vm_list=[],
                                            privileges=[])
-        if error_info:
-            err = error_code.TENANT_CREATE_FAILED.format(auth.DEFAULT_TENANT, error_info)
-            logging.warning(err)
+            if error_info:
+                logging.warning(error_info.msg)
 
         # create DEFAULT privilege if needed
-        error_info, privileges = auth.get_default_privileges()
+        error_msg, privileges = auth.get_default_privileges()
+        if error_msg:
+            logging.warning(error_msg)
+            return
+
```

Comment:
Addressed your comments. Now the logic is
1. check default_tenant exists or not
2. if default_tenant exists, do nothing, continue
3. if default_tenant does not exist, try to create, assert the create should not return error, since the test need to start with default_tenant exist

Similar logic also apply to default_privilege